src/algebra/algebra/spectrum.lean

```diff
@@ -245,6 +243,84 @@ begin
     exact ⟨unit_mem_mul_iff_mem_swap_mul.mp k_mem, k_neq⟩ },
 end
 
+open polynomial
+/-- This is half of the spectral mapping theorem for polynomials. We prove it separately
+because it holds over any field, whereas `spectrum.polynomial_spectrum` needs the field to
```

Comment:
I assume `spectrum.polynomial_spectrum` has changed name?

Reply:
Whoops, yes, sorry. Lean doesn't tell me when my documentation needs updating, and I haven't yet figured out a way to reliably remind myself to check it.